src/parsing.ts

```diff
@@ -87,31 +94,18 @@ export const learnFileId = async (_graph: Graph, filePath: string) => {
 
 export const parseDirectory = async (
   graph: Graph,
-  directory: string,
   fileCallback: (graph: Graph, path: string) => Promise<void>
 ) => {
-  const files = await vscode.workspace.fs.readDirectory(
-    vscode.Uri.file(directory)
+  const files = await vscode.workspace.findFiles(
```

Comment:
```suggestion
  // `findFiles` is used here since it respects files excluded by either the
  // global or workspace level files.exclude config option.
  const files = await vscode.workspace.findFiles(
```

I wouldn't hesitate to add a comment here giving more context on why usage of this particular function makes a difference.

Reply:
Yes, I'd thought about putting a comment. Good spot (y)